README.md

```diff
@@ -1,6 +1,6 @@
 # Chrono (v2)
 
-A natural language date parser in Javascript. 
```

Comment:
What autoformat do you use? Could you revert the style change on this file? Or submit it separately with the suggested formatter?

Reply:
Woops! My bad! :smile: -- I just format my files with `sed`, and I guess it picked up the `README.md` on accident. Reverted! I'll push it up shortly.

>```
>$ sed -i'' -E 's,[[:space:]],
>```